.travis.yml

```diff
@@ -2,7 +2,9 @@ sudo: false
 
 language: node_js
 node_js:
-  - iojs
+  - '5'
```

Comment:
Further discussion on this and the pull requests I've made in the past couple of days:
• https://github.com/postcss/autoprefixer/pull/598
• https://github.com/sindresorhus/ava/pull/388